app/src/test/java/com/kickstarter/viewmodels/PledgeFragmentViewModelTest.kt

```diff
@@ -83,13 +89,47 @@ class PledgeFragmentViewModelTest : KSRobolectricTestCase() {
         this.cards.assertValueCount(2)
     }
 
+    @Test
+    fun testContinueButtonIsGone() {
+        setUpEnvironment(environment())
+
+        this.cards.assertNoValues()
+        this.continueButtonIsGone.assertValue(false)
+        this.paymentContainerIsGone.assertValue(true)
+    }
+
+    @Test
+    fun testLoggingInUser() {
+        val mockCurrentUser = MockCurrentUser()
+        setUpEnvironment(environment().toBuilder().currentUser(mockCurrentUser).build())
+
+        this.cards.assertNoValues()
+        this.continueButtonIsGone.assertValue(false)
+        this.paymentContainerIsGone.assertValue(true)
+
+        mockCurrentUser.refresh(UserFactory.user())
+
+        this.cards.assertValueCount(1)
+        this.continueButtonIsGone.assertValues(false, true)
+        this.paymentContainerIsGone.assertValues(true, false)
+    }
+
     @Test
     fun testEstimatedDelivery() {
         setUpEnvironment(environment())
 
         this.estimatedDelivery.assertValue("March 2019")
     }
 
+    @Test
```

Comment:
Same here. I think we could either only test the `paymentContainerIsGone` output, as the method name describes, or maybe rename the `testLoggingInUser` test to something more general and delete this test and the `testContinueButtonIsGone`.